xblock/plugin.py

```diff
@@ -75,12 +110,20 @@ def _load_class_entry_point(cls, entry_point):
     def load_class(cls, identifier, default=None, select=None):
         """Load a single class specified by identifier.
 
-        If `identifier` specifies more than a single class, and `select` is not None,
-        then call `select` on the list of entry_points. Otherwise, choose
-        the first one and log a warning.
+        By default, this returns the class mapped to `identifier` from entry_points
+        matching `{cls.entry_points}.overrides` or `{cls.entry_points}`, in that order.
+
+        If multiple classes are found for either `{cls.entry_points}.overrides` or
+        `{cls.entry_points}`, it will raise an `AmbiguousPluginError`.
 
-        If `default` is provided, return it if no entry_point matching
-        `identifier` is found. Otherwise, will raise a PluginMissingError
+        If no classes are found for `{cls.entry_points}`, it will raise a `PluginMissingError`.
```

Comment:
Swap both lines as per order
```python
        If no classes are found for `{cls.entry_points}`, it will raise a `PluginMissingError`.


        If multiple classes are found for either `{cls.entry_points}.overrides` or
        `{cls.entry_points}`, it will raise an `AmbiguousPluginError`.
```

Reply:
Arguably, `load_class` doesn't even care about those details since those reside in the `select` function. I would prefer to delete them entirely than split hairs over the ordering.

Reply:
> Arguably, load_class doesn't even care about those details since those reside in the select function. 

+1, they are good as is

FWIW they are helpful lines to have in the docstring